.github/workflows/workflow_finetune.yml

```diff
@@ -68,8 +68,10 @@ jobs:
 
       - name: Build Docker Image
         run: |
-          docker build ./ --build-arg CACHEBUST=1 --build-arg http_proxy=${{ inputs.http_proxy }} --build-arg https_proxy=${{ inputs.https_proxy }} -f dev/docker/Dockerfile.cpu_and_deepspeed -t finetune:latest && yes | docker container prune && yes
-          docker image prune -f
+          source .github/workflows/config/docker.sh
+          build_and_prune "finetune" ".cpu_and_deepspeed" ${{ inputs.http_proxy }} ${{ inputs.https_proxy }}
```

Comment:
I think you can move some environment constants to .github/workflows/config/docker.sh, let the function focus on the key parameters regarding to docker.